logIn/views.py

```diff
@@ -6,18 +6,21 @@
 
 
 def login_user(request, user_type):
+    valid_error = False
     login_error = False
     if request.method == "POST":
         form = LoginForm(request.POST)
         if form.is_valid():
             username = form.cleaned_data["username"]
             password = form.cleaned_data["password"]
             if user_type == UserType.STUDENT:
-                user = Student.objects.filter(username=username, password=password)
-                if user:
-                    return HttpResponse("Yay! We do remember you... (Student)")
-                else:
+                user = Student.objects.get(username=username)
+                if not user.is_active:
+                    valid_error = True
+                elif user.password != password:
                     login_error = True
+                else:
+                    return HttpResponse("Yay! We do remember you... (Student)")
             elif user_type == UserType.ADMIN_STAFF:
                 user = Admin_Staff.objects.filter(username=username, password=password)

```

Comment:
Shouldn't we be doing the same checks for "admin_staff" user type as done for login for "student" user type?

Reply:
the checks are different because email verification has not been set up for the admissions staff